desktop/src/main/java/bisq/desktop/main/content/components/UserProfileSelection.java

```diff
@@ -291,13 +284,22 @@ protected void onViewAttached() {
                 }
             });
             comboBoxWidthPin = EasyBind.subscribe(model.comboBoxWidth, w -> comboBox.setComboBoxWidth(w.doubleValue()));
+
+            isComboBoxPin = FxBindings.subscribe(model.isPrivateChannel, isPrivate -> {
```

Comment:
I think it would be more clear if we call IsPrivateChannel field in model `useComboBox` and adopt that to the local variable and pin names.

Reply:
I think that would be more confusing, for example in this case
```
                if (model.isPrivateChannel.get() && selectedUserIdentity != null) {
                    new Popup().warning(Res.get("chat.privateChannel.changeUserProfile.warn", selectedUserIdentity.getUserProfile().getUserName()))
                            .onClose(() -> {
                                model.selectedUserProfile.set(null);
                                model.selectedUserProfile.set(new ListItem(selectedUserIdentity));
                            })
                            .show();
```

The field is describing if it's a private channel rather than if the combobox should be used.

With the name `isComboBoxPin` for the pin it's quite clear that the status as private or public channel is what dictates whether the profile should be a combobox or a text field.